theme/src/components/icons/icons.type.ts

```diff
@@ -2,3 +2,7 @@ export interface IconProps {
   className?: string;
   alt?: string;
 }
+
+export interface IconPropsColor extends IconProps {
+  color?: string;
+}
```

Comment:
This will conflict with my changes in #168, but I like this approach better so that only icons that support changing the color can use it. Although, I imagine if we get to the point where every icon has this prop, we'll be able to consolidate it with the `IconProps` interface.

One thing is I would name this `IconColorProps` to be consistent with how  prop interfaces are named and I find it more natural that `Icon` and `Color` are next to each other since that's what the interface does